### PR TITLE
Cargo value changes.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
@comma would appreciate your eyes on this to check if my logic is sound or not.

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
So the numbers this produces are a bit nuts.
````
pack cost for EVA - Atmospherics voidsuit is 6760
pack cost for Electronics - Atmospherics scanner modules is 10
pack cost for Emergency equipment is 21382.5
pack cost for Equipment - Empty gas canister is 100
pack cost for Equipment - Inflatable barriers is 1642.5
pack cost for Equipment - Rapid Piping Device is 962.5
pack cost for Gas - Air canister is 100
pack cost for Gas - Carbon dioxide gas canister is 100
pack cost for Gas - Hydrogen canister is 100
pack cost for Gas - Nitrogen canister is 100
pack cost for Gas - Nitrous oxide gas canister is 100
pack cost for Gas - Oxygen canister is 100
pack cost for Gas - Phoron gas canister is 100
pack cost for Gas - Phoron tanks is 3817.5
pack cost for Gear - Internals is 6150
pack cost for Liquid - Fuel tanks is 5090
pack cost for Equipment - Body bags is 127.5
pack cost for Gear - Cleaning supplies is 6762.5
pack cost for Gear - Janitor biohazard equipment is 3885
pack cost for Gear - Janitorial supplies is 6420
pack cost for Misc - Pest control is 22.5
pack cost for Spares - Replacement lights is 7897.5
pack cost for Reagent refill - Acetone is 1515
pack cost for Reagent refill - Ale is 1515
pack cost for Reagent refill - Aluminium is 7515
pack cost for Reagent refill - Ammonia is 7515
pack cost for Reagent refill - Beer is 1515
pack cost for Reagent refill - Cafe Latte is 765
pack cost for Reagent refill - Caramel Syrup is 765
pack cost for Reagent refill - Carbon is 7515
pack cost for Reagent refill - Chai Tea is 765
pack cost for Reagent refill - Cherry Cola is 765
pack cost for Reagent refill - Chocolate Syrup is 765
pack cost for Reagent refill - Citrus Soda is 765
pack cost for Reagent refill - Coffee is 765
pack cost for Reagent refill - Cognac is 1515
pack cost for Reagent refill - Cola is 765
pack cost for Reagent refill - Copper is 7515
pack cost for Reagent refill - Cream is 765
pack cost for Reagent refill - Ethanol is 1515
pack cost for Reagent refill - Gin is 1515
pack cost for Reagent refill - Green Tea is 765
pack cost for Reagent refill - Hot Coco is 765
pack cost for Reagent refill - Hydrazine is 1515
pack cost for Reagent refill - Hydrochloric Acid is 3015
pack cost for Reagent refill - Ice is 765
pack cost for Reagent refill - Iron is 1515
pack cost for Reagent refill - Kahlua is 1515
pack cost for Reagent refill - Lemon-Lime Juice is 765
pack cost for Reagent refill - Lemonade is 765
pack cost for Reagent refill - Lime Juice is 765
pack cost for Reagent refill - Lithium is 1515
pack cost for Reagent refill - Mead is 1515
pack cost for Reagent refill - Mercury is 1515
pack cost for Reagent refill - Milk is 765
pack cost for Reagent refill - Orange Juice is 765
pack cost for Reagent refill - Phosphorus is 1515
pack cost for Reagent refill - Potassium is 1515
pack cost for Reagent refill - Pumpkin Spice Syrup is 765
pack cost for Reagent refill - Radium is 1515
pack cost for Reagent refill - Rooibos Tea is 765
pack cost for Reagent refill - Rum is 1515
pack cost for Reagent refill - Silicon is 1515
pack cost for Reagent refill - Soda Water is 765
pack cost for Reagent refill - Sodium is 1515
pack cost for Reagent refill - Soy Latte is 765
pack cost for Reagent refill - Sugar is 1515
pack cost for Reagent refill - Sulfur is 1515
pack cost for Reagent refill - Sulfuric Acid is 1515
pack cost for Reagent refill - Tea is 765
pack cost for Reagent refill - Tequila is 1515
pack cost for Reagent refill - Tonic Water is 765
pack cost for Reagent refill - Tungsten is 1515
pack cost for Reagent refill - Vanilla Syrup is 765
pack cost for Reagent refill - Vermouth is 1515
pack cost for Reagent refill - Vodka is 1515
pack cost for Reagent refill - Water is 15
pack cost for Reagent refill - Watermelon Juice is 765
pack cost for Reagent refill - Whiskey is 1515
pack cost for Reagent refill - Wine is 1515
pack cost for EVA - Voidsuit, Engineering is 5325
pack cost for Electronics - Superconducting magnetic energy storage unit circuitry is 315
pack cost for Equipment - Binary atmospheric circulator is 370
pack cost for Equipment - Disposals pipe dispenser is 380
pack cost for Equipment - Emitter is 120
pack cost for Equipment - Field generator is 110
pack cost for Equipment - Particle accelerator is 355
pack cost for Equipment - Pipe Dispenser is 380
pack cost for Equipment - Shield generator construction kit is 435
pack cost for Equipment - Singularity generator is 55
pack cost for Gear - Electrical maintenance is 8595
pack cost for Gear - Firefighting equipment is 6947.5
pack cost for Gear - Mechanical maintenance is 11442.5
pack cost for Gear - Radiation protection gear is 12540
pack cost for Liquid - Fuel tank is 340050
pack cost for Parts - Basic superconductive magnetic coil is 112.5
pack cost for Parts - Emergency Bluespace Relay parts is 3847.5
pack cost for Parts - Robotics assembly is 4195
pack cost for Parts - Superconductive capacitance coil is 112.5
pack cost for Parts - Superconductive magnetic coil is 112.5
pack cost for Parts- Superconductive Transmission Coil is 112.5
pack cost for Power - Collector is 110
pack cost for Power - Mark I Thermoelectric Generator is 377.5
pack cost for Power - P.A.C.M.A.N. portable generator parts is 322.5
pack cost for Power - Solar assembly is 320
pack cost for Power - Solar pack is 602.5
pack cost for Power - Super P.A.C.M.A.N. portable generator parts is 322.5
pack cost for Power - Supermatter core is 55
pack cost for Blue and gold carpet is 3500
pack cost for Brown carpet is 3500
pack cost for Dark floor tiles is 67.5
pack cost for Freezer floor tiles is 62.5
pack cost for Green carpet is 3500
pack cost for Linoleum is 3500
pack cost for Orange carpet is 3500
pack cost for Pale blue carpet is 3500
pack cost for Purple carpet is 3500
pack cost for Red carpet is 3500
pack cost for Sea blue carpet is 3500
pack cost for White floor tiles is 62.5
pack cost for Bar - Bar supplies is 290
pack cost for Bar - Party equipment is 3675
pack cost for Bar - Premium drinks is 835
pack cost for Emergency - MREs is 735
pack cost for Emergency - Surprise pack of five pizzas is 562.5
pack cost for Equipment - Booze dispenser is 55
pack cost for Equipment - Soda dispenser is 55
pack cost for General - Kitchen supplies is 610
pack cost for Perishables - Beef is 45
pack cost for Perishables - Eggs is 135
pack cost for Perishables - Goat meat is 45
pack cost for Perishables - Milk is 90
pack cost for Perishables - Poultry is 45
pack cost for Perishables - Seafood is 75
pack cost for Deco - Potted plants is 1350
pack cost for Equipment - Beekeeping is 45
pack cost for Equipment - Hydroponics tray is 55
pack cost for Gear - Hydroponics Supplies is 8862.5
pack cost for Gear - Weed control is 9560
pack cost for Liquid - Water tank is 50
pack cost for Samples - Exotic seeds is 45
pack cost for Samples - Mundane Seeds is 85
pack cost for Inert - Monkey cubes is 57.5
pack cost for Inert - Spiders is 57.5
pack cost for Live - Chicken is 0
pack cost for Live - Corgi is 0
pack cost for Live - Cow is 0
pack cost for Live - Goat is 0
pack cost for Live - Goose is 0
pack cost for 10 diamond sheets is 2687.5
pack cost for 10 gold sheets is 2125
pack cost for 10 osmium carbide plasteel sheets is 1517.5
pack cost for 10 phoron sheets is 5937.5
pack cost for 10 plasteel sheets is 1330
pack cost for 10 silver sheets is 1905
pack cost for 10 slabs of marble is 1717.5
pack cost for 10 titanium sheets is 1592.5
pack cost for 10 uranium sheets is 3530
pack cost for 25 ebony planks is 6342.5
pack cost for 25 mahogany planks is 3655
pack cost for 25 walnut planks is 2250
pack cost for 25 yew planks is 1030
pack cost for 50 aluminium sheets is 967.5
pack cost for 50 glass sheets is 1155
pack cost for 50 maple planks is 2060
pack cost for 50 osmium carbide plasteel sheets is 2717.5
pack cost for 50 plasteel sheets is 2530
pack cost for 50 plastic sheets is 312.5
pack cost for 50 slabs of marble is 3717.5
pack cost for 50 steel sheets is 1625
pack cost for 50 titanium sheets is 4592.5
pack cost for 50 wooden planks is 1217.5
pack cost for EMERGENCY - Burn pouches is 3232.5
pack cost for EMERGENCY - Low oxygen pouches is 2332.5
pack cost for EMERGENCY - Toxin pouches is 1372.5
pack cost for EMERGENCY - Trauma pouches is 3952.5
pack cost for Electronics - Auto-Compressor crate is 7.5
pack cost for Electronics - Compact Defibrilator crate is 455
pack cost for Electronics - Defibrilator crate is 457.5
pack cost for Electronics - Medical scanner modules is 10
pack cost for Equipment - Body bags is 127.5
pack cost for Equipment - Portable freezers is 70
pack cost for Equipment - Rescue bags is 15
pack cost for Equipment - Roller bed crate is 30
pack cost for Equipment - Wheelchair crate is 50
pack cost for Gear - Autopsy equipment is 4320
pack cost for Gear - Chief medical officer equipment is 2062.5
pack cost for Gear - Medical Doctor equipment is 3622.5
pack cost for Gear - Medical biohazard gear is 23027.5
pack cost for Gear - Medical scrubs is 5502.5
pack cost for Gear - Medical surplus equipment is 30
pack cost for Gear - Medical uniforms is 9570
pack cost for Gear - Paramedic equipment is 6672.5
pack cost for Gear - Pharmacist equipment is 3562.5
pack cost for Gear - Psychiatrist equipment is 1907.5
pack cost for Gear - Sterile clothes is 4070
pack cost for Gear - Surgery tools is 4997.5
pack cost for Refills - Blood packs is 75
pack cost for Refills - Medical supplies is 11827.5
pack cost for Refills - Nanoblood is 2420
pack cost for Triage - Advanced burn supplies is 150
pack cost for Triage - Advanced trauma supplies is 300
pack cost for Triage - Stability kit is 3067.5
pack cost for Art - Arts and Crafts supplies is 357.5
pack cost for Art - Painting Supplies is 40
pack cost for Costume - Chaplain is 880
pack cost for Costume - Collectible hats! is 5290
pack cost for Costume - Formalwear is 1700
pack cost for Costume - Random is 3675
pack cost for Costume - Regular hats is 3535
pack cost for Costume - Witch is 615
pack cost for Costume - Wizard is 625
pack cost for Costume - Womens formal dress locker is 935
pack cost for Deco - Lava lamps is 22.5
pack cost for Deco - Office toys is 30
pack cost for Mod - "Burning Chrome" exosuit modkit is 7.5
pack cost for Mod - "Firestarter" exosuit modkit is 7.5
pack cost for Rec - Aromatherapy is 307.5
pack cost for Rec - Lasertag equipment is 10905
pack cost for Rec - Musical Instruments is 22.5
pack cost for Rec - Trading Cards is 10
pack cost for EVA - Engineering voidsuit is 5325
pack cost for EVA - Medical voidsuit is 4125
pack cost for EVA - Security (armored) voidsuit is 4950
pack cost for Equipment - Cargo Train Trolley is 2.5
pack cost for Equipment - Cargo Train Tug is 320
pack cost for Equipment - Ore box is 50
pack cost for Equipment - Plasma Cutter is 24502.5
pack cost for Gear - Webbing, vests, holsters. is 660
pack cost for Office supplies is 1785
pack cost for UNLISTED - Contraband crate is 4072.5
pack cost for Electronics - Reagent scanner modules is 10
pack cost for Equipment - Chemical Reagent dispenser is 55
pack cost for Equipment - Flood lamps is 5
pack cost for Gear - Illumination grenades is 40
pack cost for Parts - Phoron device kit is 3847.5
pack cost for Parts - Robotics is 4195
pack cost for Shaft miner equipment is 4425
pack cost for Ammunition - Beanbag shells is 2377.5
pack cost for Ammunition - Lethal shells is 4110
pack cost for Ammunition - SMG top mounted is 8420
pack cost for Ammunition - SMG top mounted practice is 3240
pack cost for Ammunition - SMG top mounted rubber is 2420
pack cost for Ammunition - military rifle is 7530
pack cost for Ammunition - military rifle practice is 3060
pack cost for Armor - Ablative is 19070
pack cost for Armor - Arm and leg guards, black is 3840
pack cost for Armor - Arm and leg guards, blue is 3840
pack cost for Armor - Arm and leg guards, green is 3840
pack cost for Armor - Arm and leg guards, navy blue is 3840
pack cost for Armor - Arm and leg guards, tan is 3840
pack cost for Armor - Ballistic is 18510
pack cost for Armor - Light is 8100
pack cost for Armor - Riot gear is 73120
pack cost for Armor - Tactical is 5845
pack cost for Armor - Unmarked is 5150
pack cost for EVA - Security (armored) voidsuit is 4950
pack cost for Equipment - Wall shield Generators is 110
pack cost for Forensics - Auxiliary tools is 1780
pack cost for Forensics - investigation equipment is 5100
pack cost for Gear - Security biohazard gear is 3977.5
pack cost for Grenades - Special Ops supplies is 1552.5
pack cost for Weapons - Electromagnetic is 4287.5
pack cost for Weapons - Energy sidearms is 7260
pack cost for Weapons - Flashbangs is 85
pack cost for Weapons - Security basic is 12840
pack cost for Weapons - Shotgun is 3030
pack cost for Weapons - Tear gas grenades is 111315
pack cost for Weapons - Energy sidearms (For disposal) is 7260
pack cost for Ammo - Armor Piercing Mass Driver Slug is 15
pack cost for Ammo - Mass Driver Slug is 15
pack cost for Ammo - disperser-EM2-QUASAR charge is 15
pack cost for Ammo - disperser-FR1-ENFER charge is 15
pack cost for Ammo - disperser-MN3-BERGBAU charge is 15
pack cost for Ammo - disperser-XP4-INDARRA charge is 15
pack cost for Cargo - Wrapping paper is 562.5
pack cost for Custodial - Toilet paper (12) is 75
pack cost for Electronics - EFTPOS scanner is 7.5
pack cost for Electronics - Paper scanner modules is 10
pack cost for Electronics - Spare PDAs is 1035
pack cost for Emergency - Canned foods is 217.5
pack cost for G.O.S.H - Barricade Tapes (mixed) is 37.5
pack cost for Liquid - Fuel tank is 340050
pack cost for Liquid - Water tank is 50
pack cost for Material - cardboard sheets (50) is 2.5
pack cost for Medium - Blank Tapes (14) is 882.5
pack cost for Refills - Bottled water is 50
pack cost for Refills - Snack foods is 307.5
pack cost for Refills - Soda cans is 225
pack cost for Refills - Toner cartridges is 22.5
pack cost for Stationery - sticky notes (50) is 5
pack cost for Vendor - BODA - Zakuson TCC is 55
pack cost for Vendor - Booze-o-mat - GrekkaTarg Boozeries is 55
pack cost for Vendor - Cigarettes - Gideon Asbestos Mining Conglomerate is 55
pack cost for Vendor - Coffee - Hot Drinks LCD is 55
pack cost for Vendor - Dinnerwares - Plastic Tat Inc is 55
pack cost for Vendor - Engineering - Dandytronics LLT is 55
pack cost for Vendor - Fitness - SwolMAX Bros is 55
pack cost for Vendor - Games - Honk Co is 55
pack cost for Vendor - Getmoore Chocolate Co is 55
pack cost for Vendor - Lavatory Essentials - Waffle Co is 55
pack cost for Vendor - Mars Mart SCC is 55
pack cost for Vendor - Nippon-tan - ArigatoRobotics LCD is 55
pack cost for Vendor - Robotics - Dandytronics LLT is 55
pack cost for Vendor - Snix Zakuson TCC is 55
pack cost for Vendor - Softdrinks Robust Industries LLC is 55
pack cost for Vendor - Tools - YouTool Co is 55
````

However none of the maps in the repo and none of the Scav maps (other than Exodus which is still in dev) actually use the cargo supply datums. I'd kinda like to merge this to get the last real holdout of arbitrary values unified with the rest of the systems so adjusting the numbers can be done once and for all.

File: code/datums/supplypacks/supplypack.dm [hunk 1]
```diff
@@ -18,6 +20,10 @@
 	if(!num_contained)
 		for(var/entry in contains)
 			num_contained += max(1, contains[entry])
+	cost = 0
+	for(var/entry in contains)
+		cost += atom_info_repository.get_worth_for(entry) * max(1, contains[entry])
+	cost *= CARGO_PRICE_MARKUP
```
Comment (afterthought2) [inline code/datums/supplypacks/supplypack.dm]: Should add the value of the crate itself. For quite a few items that's actually the bulk of the cost, even currently.

**Review by afterthought2 — DISMISSED** [review]

**Comment by MistakeNot4892** [comment]
Still need to go over these numbers - decided to do the matter standardization PR first as that will directly impact the numbers used for worth.

**Comment by MistakeNot4892** [comment]
Rolled back the actual worth/value changes for this PR, only included the cargo pricing changes. Will likely be absurd or unusable in some aspects but as none of our existing maps or downstreams use cargo I am ok with this being merged now with item worth adjusted in follow-up PRs.

**Comment by MistakeNot4892** [comment]
Split the usable parts of this into another PR.

**Pull request closed without merging**